tool/src/test/resources/filesystem-integrity/fs-unsigned-realms/fsRealmAl/a/l/alice-MFWGSY3F.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<identity xmlns="urn:elytron:1.0">
+    <attributes>
+        <attribute name="role" value="user"></attribute>
+        <attribute name="role" value="admin"></attribute>
+    </attributes>
+
+    </identity>
```

Comment:
You should run a formatter on this XML file ;-)

Reply:
Thanks @rhusar, updated.